### PR TITLE
avoid auto popup of `ensureEthereumChain`

### LINKED ISSUE #189: Bridge page tries to connect MetaMask before `Connect Wallet` is clicked

### PR DESCRIPTION
**Pull request by aidan-starke**
## Description

 - closes #189

#### Changes

 - remove `ensureEthereumChain` from `AppSwitch`
 - `ensureEthereumChain` first if connecting metamask on bridge
 - if on bridge and not connected to ethereum, offer link to switch networks
 - only fetch bridge status & verification fee if connected to ethereum

### CODE REVIEW

File: providers/MetaMaskWalletProvider.tsx [hunk 1]
```diff
@@ -31,13 +33,17 @@ const MetaMaskWalletProvider: FC<MetaMaskWalletProviderProps> = ({
...
+
 	const connectWallet = useCallback(
 		async (callback) => {
 			if (!extension) {
 				callback?.();
 				return promptInstallExtension();
 			}
 
+			if (section === "bridge") await ensureEthereumChain(extension);
+
```
Comment (ken-centrality) [inline providers/MetaMaskWalletProvider.tsx:46]: I think we should move this logic to `SubmitButton` and keep `MetaMaskWalletProvider` unaware of what section it's in

**Review by ken-centrality — CHANGES_REQUESTED** [review]
looks good, is this to address Jaffar feedbacks?

**Comment by ken-centrality** [comment]
shall we close this in favor of #195 ?

**Comment by aidan-starke** [comment]
reworked in #195

**Pull request closed without merging**